MISS_HIT.m

```diff
@@ -11,6 +11,8 @@ function mh_style(varargin)
             if ismac
                 suppress_output = '1> /dev/null 2> /dev/null';
                 set_environment = 'export PYTHONIOENCODING=UTF-8 && ';
+                % add path(substitute your own user name for rookie)
+                setenv('PATH', getenv('PATH')+":/Users/rookie/.local/bin/")
```

Comment:
Is there any chance that the username can be acquired dynamically? What's the output of system("id -un").

Reply:
done

Reply:
how to cite the revised code like u in pull request conversation

Reply:
Click one of your commit and a "+" will pop up when you move your mouse to a certain line number and you can add comments on it. 
![image](https://user-images.githubusercontent.com/10386119/143815242-cd18526f-a389-4f0f-9e99-247aef98f8dc.png)